lib/karafka/schemas/config.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module Karafka
+  # Namespace for all the validation schemas that we use to check input
+  module Schemas
+    # Regexp for validating format of groups and topics
```

Comment:
Link the regex to something like [Regex 101](http://regex101.com). Have them try it out interactively, someone will mess it up.